style-guide/about-style-guide.md

```diff
@@ -0,0 +1,168 @@
+# The style guide
```

Comment:
This starts with sentence case, then switches to title case. Since TGD's own style guide suggests Google's style guide, I think sentence case is probably the right choice here

Reply:
Yeah, I debated that. I'll go through and make these changes.